mlflow/models/evaluation/base.py

```diff
@@ -953,9 +1067,10 @@ def scatter_plot(eval_df, builtin_metrics, artifacts_dir):
                                        evaluators,
                                        custom_metrics=[squared_diff_plus_one, scatter_plot],
                                    )
-    :param validation_thresholds: (Optional) A dictionary of name to MetricThreshold used for
-                                             model validation.
-                                             Metric name must be one of the builtin metric's name or
+    :param validation_thresholds: (Optional) A dictionary of metric name to
```

Comment:
Can we add a docstring example of how `validation_thresholds` and `baseline_model` are used?